grpc/src/main/java/com/linecorp/armeria/internal/server/grpc/AbstractServerCall.java

```diff
@@ -388,7 +388,7 @@ private void invokeOnMessage(I request, boolean halfClose) {
         }
     }
 
-    final void invokeHalfClose() {
+    public final void invokeHalfClose() {
```

Comment:
```suggestion
    private void invokeHalfClose() {
```

Reply:
Done in [fa4d628](https://github.com/line/armeria/commit/fa4d628b0a113437da1d9ebb625332dad3b299be)!